File: app/src/main/java/at/shockbytes/coins/currency/PriceProxy.java
package at.shockbytes.coins.currency;

import java.util.List;

import at.shockbytes.coins.network.model.PriceConversion;
import rx.Observable;

/**
 * @author Martin Macheiner
 *         Date: 24.06.2017.
 */

public interface PriceProxy {

    Observable<List<PriceConversion>> getPriceConversions(List<CryptoCurrency> from, Currency to);

}

File: app/src/main/java/at/shockbytes/coins/currency/PriceProxy.kt
package at.shockbytes.coins.currency

import at.shockbytes.coins.network.model.PriceConversion
import io.reactivex.Observable

/**
 * @author Martin Macheiner
 * Date: 24.06.2017.
 */

interface PriceProxy {

    fun getPriceConversions(from: List<CryptoCurrency>,
                            to: Currency): Observable<List<PriceConversion>>

}
